lib/grp.gi

```diff
@@ -4401,6 +4401,12 @@ InstallMethod( GroupWithGenerators,
 function( gens )
 local G,typ;
 
+  if IsDomain(gens) then
+    Info( InfoPerformance, 1,
+    "Calling `GroupWithGenerators' on a domain is very inefficient.\n",
```

Comment:
It is only inefficient for large domains. So perhaps "... on a domain usually is very inefficient"?